test/parallel/test-net-pingpong.js

```diff
@@ -130,6 +130,6 @@ const tmpdir = require('../common/tmpdir');
 tmpdir.refresh();
 pingPongTest(common.PIPE);
 pingPongTest(0);
-pingPongTest(0, 'localhost');
+pingPongTest(0, '127.0.0.1');
```

Comment:
PIng, can we add `localhost` back? :)

Reply:
Hm, that doesn't make much sense to me as this test is clearly aimed at IPv4, because there's the second test that explicitly checks for IPv6. Furthermore, `localhost` here will resolve to two different addresses for listen and connect. But I'll do it just to see the result.

Reply:
Ok, so with `localhost` it fails. I'd really change it to IPv4 literal here because that's clearly what is intended.